group_project_v2/templates/html/stages/text.html

```diff
@@ -1,16 +1,4 @@
 {% if activity_stage.content_html %}
   {{ activity_stage.content_html|safe }}
   <hr/>
-{% endif %}
-
-{% if activity_stage.resources %}
-  <div class="group_resources">
-    <h4>Resources</h4>
-    {% for file_link in activity_stage.resources %}
-      {% if file_link.location %}
-        <div><a href="{{ file_link.location }}" title="{{ file_link.description }}" target="_blank">{{ file_link.title }}{% if file_link.file_name %} ({{ file_link.file_name }} - {{ file_link.submission_date }}){% endif %}</a></div>
-      {% endif %}
-    {% endfor %}
-  </div>
-  <hr/>
 {% endif %}
```

Comment:
Nit: Not your fault on this one, but while you're here, could you add in a newline?